utils/connection.py

```
@@ -0,0 +1,73 @@
+    )
+
+    df = conn.query(sql, ttl=0)
+    
+    return df.to_dict(orient="records")
+
+
+def updateDB(sql_list, param_list):
+    print("update Start!")
+    if os.getenv("GITHUB_ACTIONS") != "true":
+        load_dotenv()
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)